src/Meshes/mesh.ts

```diff
@@ -515,6 +515,16 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
 
         scene = this.getScene();
 
+        this._onBeforeDraw = ((isInstance: boolean, world: Matrix, effectiveMaterial?: Material) => {
+            if (isInstance && effectiveMaterial) {
+                if (this._uniformBuffer) {
+                    this.transferToEffect(world);
+                } else {
+                    effectiveMaterial.bindOnlyWorldMatrix(world);
+                }
+            }
+        }).bind(this);

```

Comment:
probably no need for the bind, but it works with it as well :-)

Reply:
It is required because of the way we call it

Reply:
As I want to avoid using an arrow function each time it is called